src/server/api/routes/users.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const usersController = require('../controllers/users.controller');
+
+/**
+ * @swagger
+ * /users:
+ *  get:
+ *    tags:
+ *    - Users
+ *    summary: Get all users
+ *    description:
+ *      Will return all users.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
+ */
+router.get('/', (req, res, next) => {
+  usersController
+    .getUsers()
+    .then((result) => res.json(result))
+    .catch(next);
+});
+
+/**
+ * @swagger
+ * /users/{ID}:
+ *  get:
+ *    tags:
+ *    - Users
+ *    summary: Get user by ID
+ *    description:
+ *      Will return single user with a matching ID.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: ID
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The ID of the module to get
+ *
+ *    responses:
+ *      200:
+ *        description: Successful request
+ *      5XX:
+ *        description: Unexpected error.
```

Comment:
I got this commet from Younas on my PR and I think its important here to. 
In users.controllers you also have 400 and 404 error - you need to add them to swagger here, because now if I put incorrect ID -  it shows response in 200 ok but message is error.

Reply:
Thank you Julia. I updated that part. Can you check again ?